tests/unit/querybuf.tcl

```diff
@@ -1,32 +1,39 @@
 start_server {tags {"querybuf"}} {
-    test "querybuf will never be resized when querybuf size less than 64k" {
+    test "query buffer will never be resized when less than 64k" {
         set orig_used [s used_memory]
 
-        # Memory will increase by more than 32k due to querybuf of new client.
+        # Memory will increase by more than 32k due to query buffer of client.
         set rd [redis_deferring_client]
         assert_morethan [expr [s used_memory] - $orig_used] 32768
 
-        after 1000
+        if {$::accurate} {
+            after 2000 ;# Check if client query buffer will be resized when client idle 2s
+        } else {
+            after 120 ;# serverCron only call clientsCronResizeQueryBuffer once in 100ms
```

Comment:
i think this may be unreliable (won't work on slow VMs).
we can change that whole block to a wait_for_condition, and put the entire test in an `if`

Reply:
Done it.
Use a new way to verify slave query buffer.